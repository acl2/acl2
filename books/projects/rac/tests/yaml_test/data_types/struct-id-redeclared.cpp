// RAC begin

struct S {
  int A;
  int A;
};

int foo() { return 0; }
