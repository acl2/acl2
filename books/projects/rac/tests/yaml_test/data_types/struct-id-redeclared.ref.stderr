struct-id-redeclared.cpp:5 (6-7): Duplicate identifier declaration (could be an enum, a variable template parameter or function) `A`
 5 |   int A;
   |       ^
note: first defined here
 4 |   int A;
   |       ^
